controllers/controllers/workloads/cfapp_controller_test.go

```diff
@@ -325,71 +340,6 @@ var _ = Describe("CFAppReconciler Integration Tests", func() {
 			})
 		})
 
-		When("routes exist and CFProcesses do not exist for the app", func() {
```

Comment:
Why is this being removed?

Reply:
Ok - this is a duplicate test and missing process creation has nothing to do with routes.